components/menu/demo/sider-current.md

```diff
@@ -1,10 +1,18 @@
 ---
 order: 2
-title: 只展开当前父级菜单
+title: 
+  zh−CN:只展开当前父级菜单
+  en-US:only to open the current parant menu
 ---
 
+## zh-CN
+
 点击菜单，收起其他展开的所有菜单，保持菜单聚焦简洁。
 
+## en-US
+
+Click the menu, get all the others closed to make the menu simple.
```

Comment:
Click the menu and you will see that all the other menus gets collapsed to keep the entire menu compact.